ocaml/xapi/xapi_globs.ml

```diff
@@ -446,6 +446,8 @@ let vgt_fence_sz = "vgt_fence_sz"
 
 let mxgpu_vgpus_per_pgpu = "vgpus_per_pgpu"
 
+let nvidia_compat_lookup_file = "/tmp/nvidia_compat_lookup"
```

Comment:
Should this filename have an extension? Like `.vgx` or similar?

Reply:
What code uses this file?

Reply:
It's not a vgx file or anything like it, but our own data. I don't think that an extension helps?

`/var/run/nonpersistent/xapi/` may be better, yes.